style.css

```diff
@@ -0,0 +1,70 @@
+#main {
+  border: 2px solid rgb(121, 125, 129);
+  margin-top: 10vh;
+  max-width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  background-color: rgba(185, 178, 178, 0.774);
+}
+
+#img-div {
+  border: 2px solid whitesmoke;
+  background-color: whitesmoke;
+  padding-top: 40px;
+  padding-bottom: 40px;
+  margin-top: 10vh;
+  max-width: 70%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: auto;
```

Comment:
I think it would be better to use the shortcut styling(one line). That will save you more time and would make your styling look better.